scripts/HFIR_4Circle_Reduction/multi_threads_helpers.py

```diff
@@ -258,9 +251,8 @@ def run(self):
                                                                                    normalization=self._normalizeType,
                                                                                    mask_ws_name=self._selectedMaskName)
             except ValueError as val_err:
-                print '[DB] Unable to integrate scan %d due to %s.' % (scan_number, str(val_err))
                 status = False
-                ret_obj = '%s.' % str(val_err)
+                ret_obj = 'Unable to integrate scan %d due to %s.' % (scan_number, str(val_err))
```

Comment:
I would write
` ret_obj = 'Unable to integrate scan {0} due to {1}.'.format(scan_number, str(val_err))`